appcues/src/main/java/com/appcues/debugger/DebuggerViewModel.kt

```diff
@@ -185,25 +187,14 @@ internal class DebuggerViewModel(
         }
     }
 
-    fun onDragging(dragAmount: Offset) {
-        _uiState.value = Dragging(dragAmount)
-    }
-
-    fun onDragEnd() {
-        _uiState.value = Idle(mode)
-    }
-
-    fun onDismiss() {
-        _uiState.value = Dismissing
+    fun transition(state: UIState) {
+        _uiState.value = state
```

Comment:
Detekt was not happy about how many functions this class had. A lot of them were simply pass-through functions to update the `_uiState`. So this new `transition` function replaces some of those and allows updated callsites to just initiate that transition as needed -- for example `onDismiss` below.